test/exec.js

```diff
@@ -107,6 +107,31 @@ if (process.version >= 'v0.11') { // this option doesn't work on v0.10
   assert.ok(shell.error());
 }
 
+// check process.env works
+assert.ok(!shell.env.FOO);
+shell.env.FOO = 'Hello world';
+result = shell.exec('echo $FOO');
```

Comment:
`cmd.exe` doesn't support variables prefixed with a `$`, what you want here is `%FOO%`.